README.md

```diff
@@ -4479,6 +4479,31 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 <summary>Explain Readiness probe</summary><br><b>
 </b></details>
 
+<details>
+<summary> What does being cloud-native mean?</summary><br><br>
```

Comment:
switch <br><br> with <br><b>

Reply:
Please explain further I am a bit confused?

Reply:
Sorry, it's the html formatting.
What I originally wrote is "switch \</summary>\<br>\<br> with \</summary>\<br>\<b>"